src/app/visit/visit.component.ts

```diff
@@ -56,4 +54,12 @@ export class VisitComponent implements OnInit {
     selectPatient(patientSelected: Patient) {
         this.patient = patientSelected;
     }
+    @HostListener('document:keydown', ['$event'])
+    handleKeyboardEvent(event: KeyboardEvent) {
+        if (event.key === 'ArrowRight') {
+            this.paginator.nextPage();
```

Comment:
I would add here a check if there is a next or previous page

Reply:
done but explain a this condition:
`if (event.key === 'ArrowRight' && this.paginator.hasNextPage())`